datasets/hate_speech_pl/README.md

```diff
@@ -0,0 +1,193 @@
+- **Paper:** http://www.qualitativesociologyreview.org/PL/Volume38/PSJ_13_2_Troszynski_Wawer.pdf
+- **Leaderboard:** [N/A]
+- **Point of Contact:** [Marek Troszyński](mtroszynski@civitas.edu.pl), [Aleksander Wawer](axw@ipipan.waw.pl)
+
+### Dataset Summary
+
+The dataset was created to analyze the possibility of automating the recognition of hate speech in Polish. It was collected from the Polish forums and represents various types and degrees of offensive language, expressed towards minorities.
+
+The original dataset is provided as an export of MySQL tables, what makes it hard to load. Due to that, it was converted to CSV and put to a Github repository. 
+
+### Supported Tasks and Leaderboards
+
+- `text-classification`: The dataset might be used to perform the text classification on different target fields, like the presence of irony/sarcasm, minority it describes or a topic. 
+- `text-scoring`: The sentiment analysis is another task which might be solved on a dataset.
+
+### Languages
+
+Polish, collected from public forums, including the HTML formatting of the text.
+
+## Dataset Structure
+
+### Data Instances
+
+The dataset consists of three collections, originally provided as separate MySQL tables. Here represented as three CSV files.
+
+```
+{
+  'id': 1,
+  'text_id': 121713,
+  'annotator_id': 1,
+  'minority_id': 72,
+  'negative_emotions': false,
+  'call_to_action': false,
+  'source_of_knowledge': 2,
+  'irony_sarcasm': false,
+  'topic': 18,
+  'text': ' <font color=\"blue\"> Niemiec</font> mówi co innego',
+  'rating': 0
+}
+```
+
+### Data Fields
+
+List and describe the fields present in the dataset. Mention their data type, and whether they are used as input or output in any of the tasks the dataset currently supports. If the data has span indices, describe their attributes, such as whether they are at the character level or word level, whether they are contiguous or not, etc. If the datasets contains example IDs, state whether they have an inherent meaning, such as a mapping to other datasets or pointing to relationships between data points.
+
+- `id`: unique identifier of the entry
+- `text_id`: text identifier, useful when a single text is rated several times by different annotators
+- `annotator_id`: identifier of the person who annotated the text
+- `minority_id`: the internal identifier of the minority described in the text
+- `negative_emotions`: boolean indicator of the presence of negative emotions in the text
+- `call_to_action`: boolean indicator set to true, if the text calls the audience to perform any action, typically with a negative emotions
+- `source_of_knowledge`: categorical variable, describing the source of knowledge for the post rating - 0, 1 or 2 (direct, lexical or contextual, but the description of the meaning for different values couldn't be found)
+- `irony_sarcasm`: boolean indicator of the present of irony or sarcasm
+- `topic`: internal identifier of the topic the text is about
```

Comment:
what are the available topics ?

Reply:
Unfortunately the paper doesn't clarify the meaning of the labels. The topic is just some sort of identifier, but its meaning is not described in anywhere. I may put a note on that in README though.